ingestion/functions/parsing/argentina/argentina.py

```diff
@@ -0,0 +1,233 @@
+import os
+import sys
+from datetime import datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+
+def convert_date(date_str: str):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+
+    The date is listed in YYYYmmdd format, but the data server API will
```

Comment:
this comment doesn't match the format you have written below.

Reply:
sorry copying error